Day2/day2base.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const wrapper = str => {
```

Comment:
Let's give this function a more meaningful name than `wrapper`.
The function name should give you an idea of what the function does.
`wrapper` doesn't tell anything except that the function wraps something